src/tests/inbound/createInbound/exportItems.test.ts

```diff
@@ -42,83 +44,195 @@ test.describe('Export all incoming items', () => {
-        await createInboundPage.createStep.isLoaded();
-      });
-  
-      await test.step('Create Stock Movement step', async () => {
-        await createInboundPage.createStep.originSelect.findAndSelectOption(
-          ORIGIN.name
-        );
-        await createInboundPage.createStep.requestedBySelect.findAndSelectOption(
-          USER.name
-        );
-        await createInboundPage.createStep.dateRequestedDatePicker.fill(TODAY);
-        await createInboundPage.createStep.descriptionField.textbox.fill(
-          DESCRIPTION
-        );
-      });
-  
-      await test.step('Go next step (Add items)', async () => {
-        await createInboundPage.nextButton.click();
-        await createInboundPage.addItemsStep.isLoaded();
-      });
-  
-      INBOUND_ID = createInboundPage.getId();
-  
-      await createInboundPage.addItemsStep.addItems(ROWS);
-  
-      await test.step('Go to next step (Send)', async () => {
-        await createInboundPage.nextButton.click();
-        await createInboundPage.sendStep.isLoaded();
-      });
-  
-      await test.step('Fill shipment fields (Send)', async () => {
-        await createInboundPage.sendStep.shipmentTypeSelect.findAndSelectOption(
-          SHIPMENT_TYPE
-        );
-        await createInboundPage.sendStep.expectedDeliveryDatePicker.fill(
-          EXPECTED_DELIVERY_DATE
-        );
-      });
-
-      await test.step('Send shipment', async () => {
-        await createInboundPage.sendStep.sendShipmentButton.click();
-        await stockMovementShowPage.waitForUrl();
-        await stockMovementShowPage.isLoaded();
-      });
+      const ROWS2 = [
+        {
+          product: {
+            productCode: PRODUCT_ONE.productCode,
+            productName: PRODUCT_ONE.name,
+          },
+          quantity: '10',
+        },
+      ];
 
+      await inboundListPage.goToPage();
+      const ifInboundListPageIsEmpty =
+        await inboundListPage.table.emptyInboundList.isVisible();
+      const ifInboundListPageIsNotEmpty =
+        await inboundListPage.table.emptyInboundList.isHidden();
+      if (ifInboundListPageIsEmpty) {
+        await test.step('Create and ship stock movement when inbound list is empty', async () => {
+          await createInboundPage.goToPage();
+          await createInboundPage.createStep.originSelect.findAndSelectOption(
+            ORIGIN.name
+          );
+          await createInboundPage.createStep.requestedBySelect.findAndSelectOption(
+            USER.name
+          );
+          await createInboundPage.createStep.dateRequestedDatePicker.fill(
+            TODAY
+          );
+          await createInboundPage.createStep.descriptionField.textbox.fill(
+            DESCRIPTION
+          );
+          await createInboundPage.nextButton.click();
+          await createInboundPage.addItemsStep.isLoaded();
+          await createInboundPage.wizzardSteps.assertActiveStep('Add items');
+
+          INBOUND2_ID = createInboundPage.getId();
```

Comment:
do you need to declare the `INBOUND2_ID` as a `let`? I don't see you are overriding this value, but maybe I am missing something